src/schemas/profiles.py

```
@@ -0,0 +1,134 @@
+            validate_name(name)
+            return name.lower()
+        except ValueError as e:
+            raise HTTPException(
+                status_code=422,
+                detail=[{
+                    "type": "value_error",
+                    "loc": ["first_name" if "first_name" in name else "last_name"],
+                    "msg": str(e),
+                    "input": name
+                }]
+            )
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)